### PR TITLE
SDK-2310 Add biometric consent flow

### PR DESCRIPTION
**Pull request by abdalmajeed-yoti**
Added `biometricConsentFlow` to the SDK config by using `withEarlyBiometricConsentFlow` or `withJustInTimeBiometricConsentFlow` methods.

### CODE REVIEW

File: src/idv_service/session/create/sdk.config.builder.js [hunk 1]
```diff
@@ -151,6 +151,37 @@ class SdkConfigBuilder {
...
+
+  /**
+   * Sets the biometric consent flow
+   *
+   * @param {string} biometricConsentFlow the biometric consent flow
+   *
+   * @returns {this}
+   */
+  withBiometricConsentFlow(biometricConsentFlow) {
+    Validation.isString(biometricConsentFlow, 'biometricConsentFlow');
```
Comment (laurent-yoti) [inline src/idv_service/session/create/sdk.config.builder.js:180]: Do we have any `oneOf` in the `Validation` ?
Feel that we could allow any value right now, and that method is by nature public.

Comment (MrBurtyyy) [reply src/idv_service/session/create/sdk.config.builder.js:180]: If you're implying validation on the value passed to this method by the Relying Business, this should be left to the backend.  It also allows older SDKs to provide new values that the backend may support, even if there isn't a "sugar" method for it.

Comment (laurent-yoti) [reply src/idv_service/session/create/sdk.config.builder.js:180]: Yup, thanks, as discussed, we shall not try to validate the value itself then.

**Review by laurent-yoti — APPROVED** [review]
Happy with the examples, let's merge.

**Pull request merged**